src/main/resources/templates/category/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
+<head ></head>
+<body class="container">
+
+<header>
+    <H1>All Categories</H1>
+</header>
+
+
+
+<table>
+    <thead>
+    <tr>
+        <th>Category Name</th>
+    </tr>
+    </thead>
+    <tr th:each="category : ${category}">
```

Comment:
If we do update the values I mentioned in the controller this would also need to update from ${category} to ${categories}

Reply:
done!